docs/source/_toctree.yml

```diff
@@ -56,6 +56,10 @@
       title: Load image data
     - local: image_process
       title: Process image data
+    - local: image_classification
+      title: Image classification
+    - local: object_detection
+      title: Object detection
```

Comment:
Perhaps a table at the end of the `image_process` page that redirects to a specific task (processing) would be better than having each task in the index to keep the index small.

Reply:
Small pref to have them in the index, this is the kind of things that users should be able to find in very few clicks IMO

Reply:
I think keeping it in the index would be more discoverable for users since not everyone may necessarily read the `image_process` page :)